Routines/Necromancer/NEC_Assist.cs

```diff
@@ -0,0 +1,2207 @@
+            return true;
+        }
+
+        private bool ShouldFrailty(out Vector3 position)
+        {
+            position = Vector3.Zero;
+            TrinityActor target = null;
+            if (!Skills.Necromancer.Frailty.CanCast())
+                return false;
+
+            if (Player.PrimaryResource < 10)
+                return false;
+
+            bool hasCurseBuff = AnyTargetWithDebuff(65f, SNOPower.P6_Necro_PassiveManager_Frailty);
+            
+            if (hasCurseBuff)
+            {
+                int intevalForCurse = CurseInterval*1000;
+
+                if (Skills.Necromancer.Frailty.TimeSinceUse < intevalForCurse) 
+                    return false;
+            }
+
+            target = BestTargetWithoutDebuff(60f, SNOPower.P6_Necro_PassiveManager_Frailty);
+            if  (!IsValidTarget(target))
+                return false;
+
+            position = target.Position;
+            return true;
+        }
+
+        private bool ShouldLeech(out Vector3 position)
+        {
+            position = Vector3.Zero;
+            TrinityActor target = null;
+            if (!Skills.Necromancer.Leech.CanCast())
+                return false;
+
+            if (Player.PrimaryResource < 10)
+                return false;
+
+            bool hasCurseBuff = AnyTargetWithDebuff(65f, SNOPower.P6_Necro_PassiveManager_Leech);
+            
+            if (hasCurseBuff)
+            {
+                int intevalForCurse = CurseInterval*1000;
+
+                if (Skills.Necromancer.Leech.TimeSinceUse < intevalForCurse) 
+                    return false;
+            }
+
+            target = BestTargetWithoutDebuff(60f, SNOPower.P6_Necro_PassiveManager_Leech);
+            if  (!IsValidTarget(target))
+                return false;
+
+            position = target.Position;
+            return true;
+        }
+
+        TrinityActor lastCSTarget = null;
+        double lastCSCast = 0;
+        protected override bool ShouldCommandSkeletons(out TrinityActor target)
+        {
+            target = null;
+
+            if (MyGetTickCount() - lastCSCast > 2000)
+                lastCSTarget = null;
+            
+            if (!Skills.Necromancer.CommandSkeletons.CanCast())
+                return false;
+
+            if (Skills.Necromancer.LandOfTheDead.TimeSinceUse < 10000)
+            {
+                if (Skills.Necromancer.LandOfTheDead.CanCast())
+                    return false;
+                
+                if (CastCSInterval !=0 && Skills.Necromancer.CommandSkeletons.TimeSinceUse < CastCSInterval*10)
+                    return false;
+            }
+            else 
+            {
+                int interval = 250;
+                if (Skills.Necromancer.LandOfTheDead.CanCast())
+                    interval = 500;
+                if (Skills.Necromancer.CommandSkeletons.TimeSinceUse < interval)
+                    return false;
+            }
+
+            if (false)
```

Comment:
This is never true, could be removed and replaced with
target = GetBestTarget(Player.Position, 60f, true);